osv/ecosystems.py

```diff
@@ -445,39 +449,54 @@ def sort_key(self, version):
   @staticmethod
   def _process_git_log(output: str) -> list:
     """Takes git log diff output,
-    finds all changes to pkgver and outputs that in an unsorted list"""
+    finds all changes to pkgver and outputs that in an unsorted list
+    """
     all_versions = set()
     lines = [
-        x for x in output.splitlines() if len(x) == 0 or x.startswith('+pkgver')
+        x for x in output.splitlines()
+        if len(x) == 0 or x.startswith(Alpine._PKGVER_ALIASES) or
+        x.startswith(Alpine._PKGREL_ALIASES)
     ]
     # Reverse so that it's in chronological order.
     # The following loop also expects this order.
     lines.reverse()
 
     current_ver = None
+    current_rel = '0'
+
+    def clean_versions(ver: str) -> str:
+      ver = ver.split(' #')[0]  # Remove comment lines
+      ver = ver.strip(' "\'')  # Remove (occasional) quotes
+      ver = ver.removeprefix('r')
+      return ver
 
     for x in lines:
       if len(x) == 0:
```

Comment:
Can we do

```python
if not line:
``` 

instead per https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations ?